src/include/txn.i

```diff
@@ -216,8 +216,8 @@ __wt_txn_resolve_prepared_op(
 
 	/* If we haven't found anything then there's an error. */
 	if (upd == NULL) {
-		WT_ASSERT(session, upd != NULL);
-		WT_ERR(WT_NOTFOUND);
+		WT_ERR_ASSERT(session, upd != NULL, WT_NOTFOUND, "Unable to"
+		    " locate update associated with a prepared operation.");
```

Comment:
We don't put trailing periods after messages because messages are usually concatenated together with separator characters, and a period in the middle of the error message looks odd. So, this is going to get the WiredTiger string associated with `WT_NOTFOUND` concatenated, so we'd get:
```
Unable to locate update associated with a prepared operation.: WT_NOTFOUND: item not found
```
and `<period><colon>` looks bad.